charts/vc-authn-oidc/values.yaml

```diff
@@ -64,6 +66,10 @@ controller:
     # variable_substitution_map.add_variable_substitution(
     #     r"\$sub_days_plus_one_(\d+)", sub_days_plus_one
     # )
+  # htmlFileOverrides:
+  #   ver_config_explorer.html: ""
+  #   verified_credentials.html: ""
+  #   wallet_howto.html: ""
```

Comment:
After speaking with @i5okie, it sounds like it might be best to provide the users with the ability to specify an existing (manually created) configmap to be mounted with the required contents. This could become something like `customHtmlConfigMapName: "my-custom-name"` and be referenced in the deployment if/when mounting the configmap.